libs/spark/CHANGELOG.md

```diff
@@ -13,25 +13,28 @@
   - Corrected height.
 - **NavBarButton**
   - Corrected height when no start/end icon given.
+- **Typography**
+  - TS interface accepts the `component` prop.
```

Comment:
![Will Ferrell as Buddy the Elf bursting into a coffee shop, subtitled "You did it!"](https://media.giphy.com/media/3otPoS81loriI9sO8o/giphy.gif)